jbpm-console-ng-generic/jbpm-console-ng-generic-forms/jbpm-console-ng-generic-forms-backend/src/main/resources/forms/DefaultProcess.ftl

```diff
@@ -0,0 +1,14 @@
+        <div class="form-content one-col">
+            <input type="hidden" name="processId" value="${process.id}"/>
+                <#if outputs?has_content>
+                    <h2>Outputs</h2><br/>
+                    <#list outputs?keys as key>
+                             <div class="form-row clearfix">
+                                <label>${key}</label>
+                                    <input type="text" name="${key}" value=""/>
+                            </div>
+
+                    </#list>
+                </#if>
+                <i>* Automatically generated form, only supports simple text strings *</i>
```

Comment:
@cristianonicolai agree, this is the template to generate a default form generated when there is no other form defined for a task / process. In a near future there will be a default form generation for the new forms and we could remove it.